credential-free/spring-cloud-azure-starter-jdbc-mysql/spring-cloud-azure-mysql-sample/README.md

```diff
@@ -0,0 +1,224 @@
+azurerm_mysql_flexible_server.mysql_server: Creating...
+...
+azurerm_mysql_flexible_server_firewall_rule.firewall_clientip: Creating...
+azurerm_mysql_flexible_database.database: Creating...
+azurerm_mysql_flexible_server_firewall_rule.firewall_clientip: Still creating... [10s elapsed]
+azurerm_mysql_flexible_database.database: Still creating... [10s elapsed]
+...
+Apply complete! Resources: 8 added, 0 changed, 0 destroyed.
+
+```
+
+You can go to [Azure portal](https://ms.portal.azure.com/) in your web browser to check the resources you created.
+
+### Create a MySQL non-admin user and grant permission
+
+run the following command:
+
+#### Run with Bash
+
+```shell
+source ./terraform/set_up_azure_ad_user.sh
+```
+
+#### Run with Powershell
+
+```shell
+terraform\set_up_azure_ad_user.ps1
+```
+
+### Export Output to Your Local Environment
+Running the command below to export environment values:
+
+#### Run with Bash
+
+```shell
+source ./terraform/setup_env.sh
+```
+
+#### Run with Powershell
+
+```shell
+terraform\setup_env.ps1
+```
+
+If you want to run the sample in debug mode, you can save the output value.
+
+```shell
+AZ_DATABASE_SERVER_NAME=...
+AZ_DATABASE_NAME=...
+AZ_MYSQL_AD_NON_ADMIN_USERNAME=...
+
+```
+
+## Run Locally
+
+### Run the sample with Maven
+
+In your terminal, run `mvn clean spring-boot:run`.
+
+```shell
+mvn clean spring-boot:run
+```
+
+### Run the sample in IDEs
+
+You can debug your sample by adding the saved output values to the tool's environment variables or the sample's `application.yaml` file.
+
+* If your tool is `IDEA`, please refer to [Debug your first Java application](https://www.jetbrains.com/help/idea/debugging-your-first-java-application.html) and [add environment variables](https://www.jetbrains.com/help/objc/add-environment-variables-and-program-arguments.html#add-environment-variables).
+
+* If your tool is `ECLIPSE`, please refer to [Debugging the Eclipse IDE for Java Developers](https://www.eclipse.org/community/eclipse_newsletter/2017/june/article1.php) and [Eclipse Environment Variable Setup](https://examples.javacodegeeks.com/desktop-java/ide/eclipse/eclipse-environment-variable-setup-example/).
+
+## Verify This Sample 
+
+1.1 Create a new "todo" item in the database.
+
+```shell
+curl --header "Content-Type: application/json" \
+    --request POST \
+    --data '{"description":"configuration","details":"congratulations, you have set up JDBC correctly!","done": "true"}' \
+    http://127.0.0.1:8080
+```
+
+1.2 Retrieve the data by using a new cURL request as follows.  
+
+```shell
+curl http://127.0.0.1:8080
+```
+
+Will return the list of "todo" items, including the item you've created, as follows: 
+
+```json
+[{"id":1,"description":"configuration","details":"congratulations, you have set up correctly!","done":true}]
+```
+
+## Clean Up Resources
+After running the sample, if you don't want to run the sample, remember to destroy the Azure resources you created to avoid unnecessary billing.
```

Comment:
`if you don't want to run the sample` -> `if you don't want to run the sample anymore`